README.md

```diff
@@ -0,0 +1,148 @@
+# Laravel Mail Export
+
+[![Latest Version on Packagist](https://img.shields.io/packagist/v/pod-point/laravel-mail-export.svg?style=flat-square)](https://packagist.org/packages/pod-point/laravel-mail-export)
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/pod-point/laravel-mail-export/run-tests?label=tests)
+[![Software License](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square)](LICENSE.md)
+[![Total Downloads](https://img.shields.io/packagist/dt/pod-point/laravel-mail-export.svg?style=flat-square)](https://packagist.org/packages/pod-point/laravel-mail-export)
+
+This package can export any mail sent with Laravel's `Mailable` class to any desired filesystem disk and path as a `.eml` file.
```

Comment:
Didn't look at that but can double check.

I know for sure that it won't be compatible with the legacy way of sending emails
```php
Mail::send('some.view', function ($message) {
    return $message->to('foo@example.com')
        ->from('bar@example.com')
        ->subject('Foo Bar');
});
```
because that would involve extending/overriding the `Illuminate\Mail\Message` class which is quite core, similar to the `Mailer` class.

This is why we're saying on the tin that this is for "Mailables".

Reply:
It does work with notifications as long as you use a Mailable in the `toMail()` method:

```php
public function toMail($notifiable)
{
    return (new ExportableMailable)->to($notifiable->email);
}
```

It doesn't work with `MailMessage` though and to me, it sounds like if we want to export Notifications, it's out of scope here and would require more work as it's a completely different part of the framework.

I looked at the `MailMessage` class but it's just a data object to feed the `MailChannel` for the Notification. If we wanted to be able export Notifications, we would have to consider all types too: email, sms, database, push notifications...